Source/Document Structure/SvgFragment.cs

```diff
@@ -298,16 +293,14 @@ public override SvgElement DeepCopy<T>()
         }
 
         // Override the default behavior, writing out the namespaces.
-        protected override void WriteStartElement(XmlTextWriter writer)
+        protected override void WriteStartElement(XmlWriter writer)

```

Comment:
It also adds namespace to inner SVG.(existing issue...)
This process should be in `SvgDocument`.

Reply:
Fix #702.